ClientLib/src/androidTest/java/org/droidplanner/services/android/impl/mock/MockMAVLinkClient.java

```diff
@@ -8,7 +8,6 @@
 import com.o3dr.services.android.lib.model.ICommandListener;
 
 import org.droidplanner.services.android.impl.communication.model.DataLink;
-import org.droidplanner.services.android.impl.communication.service.MAVLinkClient;
```

Comment:
Is this correct? It seems like I can build Dronekit fine right now, but it seems like this class should have this import